packages/cli/src/commands/dev.ts

```diff
@@ -40,7 +54,7 @@ export async function devCommand(options: DevCommandOptions) {
   const studioSpinner = ora('Starting local studio server').start();
   await httpServer.listen();
 
-  studioSpinner.succeed(`Novu Studio started: ${httpServer.getStudioAddress()}`);
+  studioSpinner.succeed(`Novu Studio started:\t${httpServer.getStudioAddress()}`);
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** ‏ Should this still be studio? maybe also reference it as dashboard

Reply:
I'm thinking Studio is a useful name for the locally running web thing, Dashboard is the Cloud web thing. So I'm thinking to keep as it is.